pyproject.toml

```diff
@@ -95,7 +95,7 @@ path = "mesa/__init__.py"
 # Hardcode to Python 3.11.
 # Reminder to update mesa-examples if the value below is changed.
 target-version = "py311"
-extend-exclude = ["docs", "build"]
+extend-exclude = ["build"]
```

Comment:
This addresses the large package size right? How much does it help?

Reply:
Do you mean line-length or am I missing something? 

This put docs through the ruff linter and formatter, but I also ignored a lot (see per-file-ignores)

To address the line-length across Mesa we need to add to E501 to [tool.ruff.lint] select  and if we want to be compliant with pep 8 we would set the line-length to 79. I am trying to figure out why I am still get a discrepancy between the pre-commit on my machine and the pre-commit in github actions.

Reply:
Sorry I confused this with another comment, you're right.

Personally I won't worry too much about this.